src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -120,11 +120,21 @@ public boolean isUserRegistered(String username) {
     return false;
   }
 
+  public int getNumUsers() {
+    int count = 0;
```

Comment:
This can all be simplified to:

`return users.size();`

Please also delete the print statement.